hooks/tk-multi-publish2/maya/publish_lod_geometry.py

```diff
@@ -94,6 +99,47 @@ def settings_schema(self):
         }
         return schema
 
+    def _rename_abc_top_group(self, abc_file, rename_to):
+        """
+        This function renames the "top" group of the alembic heirarchy.
+        Please dont confuse this top with the alembic root, which will always be named 'ABC'
+        The "top" here, is the first group in heirarchy of exported scene.
+        NOTE: This function does not recursively go through the entire scene and only renames the first group encountered in the chain.
+
+        :param string abc_file: Path of the alembic file to be modified.
+        :param string rename_to: The new name that you would want to assign to the top group.
+        """
+        import tempfile
+
+        if not os.path.exists(abc_file):
+            self.logger.error("Invalid Alembic path provided. Aborting the process "
+                              "-- {0}".format(abc_file)
+                              )
```

Comment:
Give an error message like `"Invalid alembic path: {}. Not renaming the top group"`.
What is process is evident while reading the code, but not when a user is just running publish.